src/Domain/Stager.php

```diff
@@ -38,21 +40,43 @@ public function __construct(Filesystem $filesystem, ProcessFactory $processFacto
     }
 
     /**
-     * @param string[] $command
+     * @param string[] $composerCommand
```

Comment:
If the command parts are expected to already be escaped, we should probably mention that in the comment.

Reply:
That's a good idea; I'll do that. (Incidentally, I've been putting off a lot of docblock comments because I don't have a clear rule for when to use them and when not to. I'd really like to come up with one and then enforce it with phpcs.